Public/Src/Engine/UnitTests/Scheduler.IntegrationTest/DirectorySymlinkTests.cs

```diff
@@ -349,6 +350,30 @@ void ValidateProducedLayout()
             }
         }
 
+        [Feature(Features.Symlink)]
+        [Feature(Features.OpaqueDirectory)]
+        [TheoryIfSupported(requiresSymlinkPermission: true)]
+        public void DirectorySymlinksInOutputDirectoryShouldSuccess()
+        {    
+            AbsolutePath rootDirAbsPath = CreateUniqueObjPath($"{SealDirectoryKind.SharedOpaque}.framework");
+            string rootDir = rootDirAbsPath.ToString(Context.PathTable);
+            System.Diagnostics.Debugger.Launch();

```

Comment:
```suggestion
```

Reply:
Remove launching debugger.

---
In reply to: [283067785](https://github.com/microsoft/BuildXL/pull/295#discussion_r283067785) [](ancestors = 283067785)